lib/pr_checker.js

```diff
@@ -211,6 +252,36 @@ class PRChecker {
     // 3. is not authored by the people opening the PR
     return true;
   }
+
+  checkCommitsAfterReview(commit, review, logger) {
+    const commitIndex = commit.length - 1;
+    const reviewIndex = review.length - 1;
+    const lastCommit = commit[commitIndex].commit;
+    const lastReview = review[reviewIndex];
+
+    const commitDate = lastCommit.committedDate;
+    const reviewDate = lastReview.publishedAt;
+
+    if (commitDate < reviewDate) {
+      logger.info('No new commits after last review');
```

Comment:
I think we can just skip this part, only warn when there is new commits, otherwise keep the noise to the minimum.